### PR TITLE
fix(learn): update seed code for D3.js challenge

### LINKED ISSUE #40685: Change the "Only change code below/above line" instructions for Data Visualization with D3: Change the Presentation of a Bar
**Describe your problem and how to reproduce it:**

Hi there!
This challenge asks users to first, add a margin of 2px to the bar class in the style tag, and next change the callback function in the ```style()``` method.
The ```//Only change code below/above line``` instructions imply that your code should only be between those two lines.
However, in order to pass the challenge, you have to edit the code below the ```// Change code above this line``` in the ```style()``` method.
I think it would be best to change those comments and replace them.

```
// Only change code below this line

// Only change code above this line
```

replace them for

```
// Change code below this line

// Change code above this line
```

and
```
// Change this line
```

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

Screenshot_2021-01-12 freeCodeCamp org

### PR DESCRIPTION
**Pull request by lucassorenson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40685

As the issue states, this is just changing some of the comments in the challenges seed code to make the solution a bit clearer. 

I know that this issue did not necessarily call for it, but I did test it on my local environment.

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
Hey there,

Thank you, for opening this PR. For future reference, please look out more for the issues with the _help wanted_ labels, as these are the issues which have been triaged, and a course-of-action has been determined. Specifically, with this, we are keeping careful watch over the comments being used, as they need to be translated.

@RandellDawson Which exactly are the pre-approved comments to be used?

**Comment by nhcarrigan** [comment]
```json
  {
    id: 'axnbgg',
    text: 'Change code above this line'
  },
  {
    id: 'i2kck7',
    text: 'Change code below this line'
  },
  {
    id: 'dlbobn',
    text: 'Change this line'
  },
  ```
  
  All three of these proposed comments are present in our dictionary. 🙂

**Comment by lucassorenson** [comment]
@ShaunSHamilton Sorry, I didn't realize that! I will look out for those tags in the future.

**Comment by ShaunSHamilton** [comment]
@lucassorenson Turns out, there are no changes needed 👍 

Thanks, @nhcarrigan

**Review by martinmilani — DISMISSED** [review]

**Review by moT01 — COMMENTED** [review]
I'm fine with this as-is, but I'm wondering if the comments should be `Add your code above/below this line` since they are adding code and not changing it. Pretty minor, the message gets across either way.

**Comment by martinmilani** [comment]
> 
> 
> I'm fine with this as-is, but I'm wondering if the comments should be `Add your code above/below this line` since they are adding code and not changing it. Pretty minor, the message gets across either way.

It seems like a more accurate approach

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.md [hunk 1]
```diff
@@ -113,10 +113,10 @@ assert(
   .bar {
     width: 25px;
     height: 100px;
-    /* Only change code below this line */
+    /* Change code below this line */
```
Comment (nhcarrigan) [inline curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.md:116]:
```suggestion
    /* Add your code below this line */
```

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.md [hunk 2]
```diff
@@ -113,10 +113,10 @@ assert(
...
 
     
-    /* Only change code above this line */
+    /* Change code above this line */
```
Comment (nhcarrigan) [inline curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-the-presentation-of-a-bar-chart.md:119]:
```suggestion
    /* Add your code above this line */
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Creating suggestions, per the discussion above.

**Comment by nhcarrigan** [comment]
@lucassorenson have you had a chance to review the suggestions above?

**Comment by lucassorenson** [comment]
@nhcarrigan Hey, sorry, I haven't checked back since I saw the "No further changes required" comment, so I didn't see the rest. I can go change that later today if it still needs to get done.

**Comment by nhcarrigan** [comment]
Yes please - you can commit the suggestions directly through GitHub by switching the PR to `files` view, selecting `add suggestion to batch` for each relevant suggestion, then `commit suggestions`.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**